.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation
+about: Suggest better documentation coverage for a particular tool or process.
+labels: "a-Docs"
+---
+
+<!--
+  Before opening a new issue, please search existing issues:  https://github.com/reposense/RepoSense/issues
+-->
+
+**What problem(s) did you run into that caused you to request additional documentation? What, if any, existing documentation relates to this proposal?**
+<!--
+    Summarize the suggested addition to or modification of existing documentation. 
+-->
+
+
+**Why should we document this and who will benefit from it?**
+<!--
+    Here would be a good place to talk about the motivation behind adding this documentation.
+-->
```

Comment:
Maybe we can also include a question regarding what does he want us to document? The given questions will imply what the person wants but better to ask explicitly?

Reply:
Alright. So the new question can be something like -
**What would you like added or modified in the documentation?**

In that case, the prompt for the first question can be changed to -
```adoc
<!--
     Mention the part of the documentation you're referring to and how it relates to your request. 
 -->
```